_examples/xkcdsearch/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: '3.5'
+services:
+  search_api:
+    container_name: 'xkcd-search'
+    build: .
+    restart: 'on-failure'
+    ports:
+      - '8000:8000'
+    depends_on:
+      - elasticsearch
+  elasticsearch:
+    container_name: 'es01'
+    image: 'bitnami/elasticsearch:7'
```

Comment:
Could you change this in favor of the official docker repository?
IE: `docker.elastic.co/elasticsearch/elasticsearch:8.1-SNAPSHOT`

Reply:
Looks perfect, thank you!